pipelines/api-build.yml

```diff
@@ -131,7 +131,7 @@ stages:
     envGroupName: '$(globalPrefix)-$(fullAppName)-api-${{ variables.env }}'
     serviceConnection: '${{ variables.nonProdServiceConnection }}'
     containerRegistry: '${{ variables.containerRegistryName }}' 
-    aspNetCoreEnvironment: 'Development'
+    aspNetCoreEnvironment: 'Test'
```

Comment:
riktig at Development skal erstattes med Test, og ikke i tillegg?

Reply:
In the "Test" stage - yes. It is still "Development" in "Dev".